mlir/test/Dialect/Vector/invalid.mlir

```diff
@@ -766,6 +766,16 @@ func.func @extract_strided_slice(%arg0: vector<4x8x16xf32>) {
   %1 = vector.extract_strided_slice %arg0 {offsets = [2], sizes = [2], strides = [1]} : vector<4x8x16xf32> to vector<3x1xf32>
 }
 
+// -----
+
+ func.func @extract_strided_slice() -> () {
+  // expected-error@+1 {{expected input vector rank to match target shape rank}}
+  %0 = arith.constant dense<1.000000e+00> : vector<24x2x2xf32>
```

Comment:
IIUC, this could be an argument? Lets reduce the number of Ops in this test to the required minimum.